<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabFlights">
       <attribute name="title">
        <string>Flights</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSimulator">
       <attribute name="title">
        <string>Simulator</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabAirports">
       <property name="windowTitle">
        <string>MainWindow</string>
       </property>
       <attribute name="title">
        <string>Airports</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabAircrafts">
       <attribute name="title">
        <string>Aircrafts</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCrew">
       <attribute name="title">
        <string>Crew</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCompanies">
       <attribute name="title">
        <string>Companies</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabLogFunctions">
       <attribute name="title">
        <string>LogFunctions</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabMisc">
       <attribute name="title">
        <string>Miscellaneous</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
